templates/contacts/contactimport_create.haml

```diff
@@ -8,6 +9,11 @@
   .flex.mb-4.flex-col(style="min-height:41px")
     -blocktrans trimmed
       You can import contacts from an Excel spreadsheet or comma separated values file.
+    -if can_validate_upload
+      -blocktrans trimmed
+        You can import contacts from an Excel spreadsheet or comma separated values file. Using the option at the end, you will have 2 groups, one containing Mobile and VOIP (contacts that you can reach using SMS) and one with landlines (contacts that you can only reach calling).
```

Comment:
I mean text `You can import contacts from an Excel spreadsheet or comma separated values file.` may appear two times.